src/views/preview/preview.jsx

```diff
@@ -106,9 +111,25 @@ class Preview extends React.Component {
     }
     addEventListeners () {
         window.addEventListener('popstate', this.handlePopState);
+        window.addEventListener('orientationchange', this.setScreenFromOrientation);
     }
     removeEventListeners () {
         window.removeEventListener('popstate', this.handlePopState);
+        window.addEventListener('orientationchange', this.setScreenFromOrientation);
+    }
+    setScreenFromOrientation () {
+        /*
+        * If the user is on a mobile device, switching to
+        * landscape format should make the fullscreen mode active
+        */
+        const isMobileDevice = screen.height <= frameless.mobile || screen.width <= frameless.mobile;
+        if (this.props.playerMode && isMobileDevice) {
+            if (screen.orientation.type === 'landscape-primary') {
```

Comment:
`screen.orientation` doesn't look like it has good browser support, have you looked at if this crashes on IE or Edge?  Is there a cross-platform way to detect the same thing?

Reply:
Yeah I could check if screen.height is smaller than screen.width.